### DISCUSSION TITLE
Problem with renewing Student pack

### DISCUSSION THREAD

**Discussion by clarissefg** [community/community#51744] [GitHub Education]
Hi everyone,

I have an issue with renewing my student pack and I've opened a ticket for this issue but it was closed without being resolved. I don't have find any help on the GitHub community. I need someone to help me with this issue but i can't find any answer or anyone and it's been 1 month now..

On the picture, you can see that I cannot click on "Continue".

Thank you for your replies

Clarisse

<img width="931" alt="image" src="https://user-images.githubusercontent.com/100565639/229350592-553b013c-6501-4738-82aa-bf810150e1e9.png">

**Comment by Ntshangase** [comment]
Hey, 

Did you try signing out then in on GitHub, maybe there was a glitch somewhere.

**Reply by clarissefg** [reply]
Hi, thank you for you reply !

I've tried this many times and it's still doesn't work.. I can't renew my student pack

**Comment by samus-aran** [comment]
Hi @clarissefg 👋 

I can't confirm but I suspect there could be an error with your email address you are trying apply with. Is there an indication that you must have a particular type of academic email to apply to that school?

**Reply by clarissefg** [reply]
Hi, thank you for you reply !

I've subscribed with this e-mail address last year and it worked fine (it also worked for my classmates). I haven't changed the school so i don't think it's an issue. And still, it doesn't work

**Comment by clarissefg** [comment]
Hi Ashley !

The email I have used for GitHub is the one my school gave to me when I applied. It’s my academic email. And this email worked last year for the Canva student pack. I don’t know if that helps :)

Best regards,

Clarisse Fatel-Gillet
5e année International Communication
06.32.70.11.44

________________________________
De : Ashley Nicolson ***@***.***>
Envoyé : Thursday, April 6, 2023 8:40:18 PM
À : community/community ***@***.***>
Cc : GILLET Clarisse ***@***.***>; Mention ***@***.***>
Objet : Re: [community/community] Problem with renewing Student pack (Discussion #51744)


Hi @clarissefg<https://github.com/clarissefg> 👋

I can't confirm but I suspect there could be an error with your email address you are trying apply with. Is there an indication that you must have a particular type of academic email to apply to that school?
